### PR TITLE
Fix alert condition when the workspace is starting

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Fix alert condition for when the workspace is starting

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/20956

### CODE REVIEW

**Comment by akurinnoy** [comment]
I've tested this PR (Che nightly, dashboard image from this PR, devworkspaces enabled) and still had this alert, the workspace started successfully afterward and the dashboard opened the IDE.

Screenshot 2021-12-22 at 12 29 15

File: packages/dashboard-frontend/src/containers/IdeLoader.tsx [hunk 1]
```diff
@@ -310,7 +310,11 @@ class IdeLoaderContainer extends React.PureComponent<Props, State> {
...
         currentStep: LoadIdeSteps.START_WORKSPACE,
       });
-    } else if (workspace?.isStopped && this.state.currentStep !== LoadIdeSteps.INITIALIZING) {
+    } else if (
+      workspace?.isDevWorkspace &&
+      workspace?.isStopped &&
+      this.state.currentStep !== LoadIdeSteps.INITIALIZING
+    ) {
       this.showAlert({
         title: `Workspace "${this.workspaceName}" is failed to start.`,
```
Comment (dkwon17) [inline packages/dashboard-frontend/src/containers/IdeLoader.tsx:319]:
```suggestion
        title: `Workspace "${this.workspaceName}" has failed to start.`,
```
Not related to this PR, but this change would be great

Comment (olexii4) [reply packages/dashboard-frontend/src/containers/IdeLoader.tsx]: I have fixed it

**Comment by olexii4** [comment]
@akurinnoy I have fixed it

**Comment by dmytro-ndp** [comment]
[dashboard-ci-test]

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**